example/session/src/main/java/org/apache/iotdb/VectorSessionExample.java

```diff
@@ -178,6 +180,23 @@ private static void selectWithLastTest()
     dataSet.closeOperationHandle();
   }
 
+  private static void createAlignedTimeseries()
+      throws StatementExecutionException, IoTDBConnectionException {
```

Comment:
How about changing the class name to `AlignedTimeseriesSessionExample`?

Reply:
fixed~